sound/soc/codecs/Kconfig

```diff
@@ -1109,6 +1110,12 @@ config SND_SOC_RT1308_SDW
 	depends on I2C && SOUNDWIRE
 	select REGMAP_SOUNDWIRE
 
+config SND_SOC_RT1316_SDW
+	tristate "Realtek RT1316 Codec - SDW"
+	depends on SOUNDWIRE
+	select REGMAP_SOUNDWIRE
+	select REGMAP_SOUNDWIRE_MBQ
```

Comment:
@shumingfan one more comment though: I don't see any MBQ usage in this patch?

Reply:
I will remove "select REGMAP_SOUNDWIRE_MBQ" for RT1316.